### DISCUSSION TITLE
Framework cloud based with GPU

### DISCUSSION THREAD

**Discussion by yamrodest** [NVlabs/sionna#192] [Q&A]
Hi!

Currently I do not have a powerful GPU to speed up the simulations. I wonder if Nvidia provides a cloud-based framework with GPUs available to run some tests/simulations.

Thank you in advance.

**Comment by kpasad** [comment]
I explored a bunch of stuff. The best I could find is vastai. They have prebuilt image with nvidia drivers. I set up conda environment and could run Sionna.
